lib/hammer_cli/output/adapter/table.rb

```diff
@@ -1,5 +1,6 @@
 require 'table_print'
 require File.join(File.dirname(__FILE__), 'wrapper_formatter')
+require File.join(File.dirname(__FILE__), '../../../table_print/formatter')
```

Comment:
I did some testing and the file `formatter.rb` is actually not being found at all. I'd move it from `lib` somewhere to 'lib/hammer_cli' and then just do
```ruby
require 'hammer_cli/...'
```